docs/migrations/ibc-migration-043.md

```diff
@@ -91,3 +91,11 @@ REST routes are not supported for these proposals.
 ## Proto file changes
 
 The gRPC querier service endpoints have changed slightly. The previous files used `v1beta1`, this has been updated to `v1`.
+
+## IBC callback changes
+
+### OnRecvPacket
```

Comment:
Big fan of this doc 👍

Reply:
Me too! We should continue this for future major version releases